oca_port/migrate_addon.py

```diff
@@ -155,12 +181,46 @@ def run(self):
                 g.run_pre_commit(self.app.repo, self.app.addon)
             else:
                 adapted = self._apply_code_pattern()
+
+            renamed = self._rename_module()
         # Check if the addon has commits that update neighboring addons to
         # make it work properly
         PortAddonPullRequest(self.app, push_branch=False).run()
-        self._print_tips(adapted=adapted)
+        self._print_tips(adapted=adapted, renamed=renamed)
         return True, None
 
+    def _rename_module(self):
+        if self.app.rename_to:
+            try:
+                import git_filter_repo as fr
```

Comment:
I remember a command that was in the OCA documentation in the past to rework history to rename a module, I can't find it anymore...
I tend to reduce the dependencies if possible, especially if a git command can do the job.

Keep this lib for now, I still didn't test your PR :)

And for such feature (renaming a module), we can consider it as required, so such dependency should be in the `pyproject.toml` (and `try/except` block can be removed).

Reply:
you mean: `git filter-branch --tree-filter 'if [ -d old_module_name ]; then mv old_module_name new_module_name; fi' HEAD` , right?

Now I am thinking git-filter-repo could be overkill for this case, let me experiment with the `git filter-branch`